### PR TITLE
Created and updated docs for 1.19-20

### LINKED ISSUE #1023: Update docs for 1.19-20

### PR DESCRIPTION
**Pull request by kschumy**
Created and updated docs for 1.19-20

closes #1023 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/contents/index.md [hunk 1]
```diff
@@ -166,6 +166,7 @@ if you are interested in the AL2 container runtime kernel version.
 * [v1-20-eks-1](releases/1-20/1/index.md) (May 18, 2021)
 
 #### EKS-D 1.19 Version Dependencies
+* [v1-19-eks-20](releases/1-19/20/index.md) (June 21, 2022)
```
Comment (kschumy) [inline docs/contents/index.md:169]:
```suggestion
* [v1-19-eks-20](releases/1-19/20/index.md) (June 20, 2022)
```

File: docs/contents/releases/1-19/20/release-announcement.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Amazon EKS Distro v1.19-20 is now available. (IMPORTANT: support for 1.19 will be ending soon. Please update to a supported version asap) This release includes an update to Amazon Linux 2 that contains the latest security fixes, and updates to some project versions and project dependencies. Amazon EKS Distro v1.19-20 builds are available through ECR Public Gallery (https://gallery.ecr.aws/eks-distro) and GitHub (https://github.com/aws/eks-distro)
```
Comment (kschumy) [inline docs/contents/releases/1-19/20/release-announcement.txt:1]:
```suggestion
Amazon EKS Distro v1.19-20 is now available. (IMPORTANT: support for 1.19 will be ending soon. Please update to a supported version asap) This release includes an update to Amazon Linux 2 that contains the latest security fixes and updates to some project versions and project dependencies. Amazon EKS Distro v1.19-20 builds are available through ECR Public Gallery (https://gallery.ecr.aws/eks-distro) and GitHub (https://github.com/aws/eks-distro)
```

File: docs/contents/releases/1-19/20/release-announcement.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+Amazon EKS Distro v1.19-20 is now available. (IMPORTANT: support for 1.19 will be ending soon. Please update to a supported version asap) This release includes an update to Amazon Linux 2 that contains the latest security fixes and updates to some project versions and project dependencies. Amazon EKS Distro v1.19-20 builds are available through ECR Public Gallery (https://gallery.ecr.aws/eks-distro) and GitHub (https://github.com/aws/eks-distro)
```
Comment (TerryHowe) [inline docs/contents/releases/1-19/20/release-announcement.txt:1]:
```suggestion
Amazon EKS Distro v1.19-20 is now available. (IMPORTANT: support for 1.19 will be ending soon. Please update to a supported version as soon as possible) This release includes an update to Amazon Linux 2 that contains the latest security fixes and updates to some project versions and project dependencies. Amazon EKS Distro v1.19-20 builds are available through ECR Public Gallery (https://gallery.ecr.aws/eks-distro) and GitHub (https://github.com/aws/eks-distro)
```

**Review by markapruett — COMMENTED** [review]
/lgtm

**Pull request merged**